graphql_compiler/macros/__init__.py

```diff
@@ -104,7 +122,25 @@ def get_schema_with_macros(macro_registry):
     Returns:
         GraphQLSchema with additional fields where macroe edges can be used.
     """
-    raise NotImplementedError()  # TODO(bojanserafimov) implement
+    schema_ast = parse(print_schema(macro_registry.schema_without_macros))
```

Comment:
I'd add a comment explaining what this line is doing. If we hadn't discussed it in person yesterday, I would have been very confused right now.